R/opts.R

```diff
@@ -588,103 +563,63 @@ opt_lineless <- function(data) {
 #'     title = "The title of the table",
 #'     subtitle = "The table's subtitle"
 #'   ) %>%
-#'   opt_add_table_outline()
+#'   opt_table_outline()
 #'
 #' @section Figures:
-#' \if{html}{\figure{man_opt_add_table_outline_1.svg}{options: width=100\%}}
+#' \if{html}{\figure{man_opt_table_outline_1.svg}{options: width=100\%}}
 #'
 #' @family Table Option Functions
 #' @section Function ID:
-#' 9-8
+#' 9-6
 #'
 #' @export
-opt_add_table_outline <- function(data,
-                                  style = "solid",
-                                  width = px(3),
-                                  color = "#D3D3D3") {
+opt_table_outline <- function(data,
+                              active = TRUE,
```

Comment:
Remove `active = TRUE`, instead the way to remove outline will be `style = "none"`

Reply:
Now done. And the other args named `"active"` now have better names.